lib/webtask.js

```diff
@@ -665,6 +680,39 @@ exports.ensure_authenticated = function(ctx) {
                         else if (req.claims.pctx && req.claims.pctx['webtask_' + claim])
                             req.claims[claim] = req.claims.pctx['webtask_' + claim];
                     });
+
+                    if(ectx && ectx.WEBTASK_JWT_SECRET && ectx.WEBTASK_JWT_AUD) {
+                        if(!req.headers['authorization'] && !req.query.key) {
+                            msg = { code: 401, message: 'must provide authorization header or key query', req_id: req.req_id };
```

Comment:
Can you make this error a bit more descriptive, e.g. "Rejecting webtask request that requires authentication. You must provide authenticating JWT token using `key` query parameter or the Authorization header."